### DISCUSSION TITLE
Problem installing AdGuard Home with RaspAP running in Bridged AP Mode

### DISCUSSION THREAD

**Discussion by Marky0** [RaspAP/raspap-webgui#1344] [General]
I understood from the [FAQ](https://docs.raspap.com/faq/#can-i-integrate-raspap-with-adguard-home) that ADGuard home was the preferred DNS AD Blocker when running RaspAP. However I am running in Bridged AP mode, and when I install AdGuard, dnsmasq is running as an active process in the background. I presume this has be started by Bridge AP mode to route DNS requests to the home router ? I am then stumped as to how I configure ADGaurd as port 53 (DNS Server) is already in use.
This is verfied by **sudo lsof -i :53**

COMMAND  PID    USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
dnsmasq 1557 dnsmasq    6u  IPv4  15833      0t0  UDP *:domain 
dnsmasq 1557 dnsmasq    7u  IPv4  15834      0t0  TCP *:domain (LISTEN)
dnsmasq 1557 dnsmasq    8u  IPv6  15835      0t0  UDP *:domain 
dnsmasq 1557 dnsmasq    9u  IPv6  15836      0t0  TCP *:domain (LISTEN)

Can someone confirm that RaspAP is running dnsmasq in Bridged AP mode, and is there anyway to get around this ?

**Comment by billz** [comment]
You can safely shutdown and disable dnsmasq in this case.
```
sudo systemctl stop dnsmasq.service 
sudo systemctl disable dnsmasq.service 
```

**Reply by Marky0** [reply]
Thanks, that seemed to work
